docs/image_summaries.ipynb

```diff
@@ -154,6 +154,7 @@
         "import matplotlib.pyplot as plt\n",
         "import numpy as np\n",
         "import sklearn.metrics\n",
+        "import tensorflow as tf\n",
```

Comment:
Maybe put this above `from tensorflow import keras` just to group the related imports?

Same goes for the next one as well.

Reply:
Done for both